lldb/source/Plugins/SymbolLocator/DebugSymbols/SymbolLocatorDebugSymbols.cpp

```diff
@@ -1066,11 +1066,21 @@ bool SymbolLocatorDebugSymbols::DownloadObjectAndSymbolFile(
   command << lookup_arg;
 
   // Log and report progress.
+  std::string lookup_desc;
+  if (uuid_ptr && file_spec_ptr)
+    lookup_desc =
+        llvm::formatv("{0} ({1})", file_spec_ptr->GetFilename().GetString(),
+                      uuid_ptr->GetAsString());
+  else if (uuid_ptr)
+    lookup_desc = uuid_ptr->GetAsString();
+  else if (file_spec_ptr)
+    lookup_desc = file_spec_ptr->GetFilename().GetString();
+
   Log *log = GetLog(LLDBLog::Host);
   LLDB_LOG(log, "Calling {0} with {1} to find dSYM: {2}", dsymForUUID_exe_path,
-           lookup_arg, command.GetString());
+           lookup_desc, command.GetString());
 
-  Progress progress("Downloading symbol file", lookup_arg);
+  Progress progress("Downloading symbol file", lookup_desc);
```

Comment:
I think this needs to say `Downloading symbol file for` now, because otherwise it sounds like we're downloading `file_spec`?

Reply:
The 3 possible status messages are:

```
Downloading symbol file: libfoo.dylib (00000000-0000-0000-0000-000000000000)
Downloading symbol file: 00000000-0000-0000-0000-000000000000
Downloading symbol file: libfoo.dylib
```

For when we have both the file and UUID, only the UUID or only the file.

Reply:
Right, this makes it sounds as if we are downloading `libfoo.dylib`. But we're downloading the symbol file for `libfoo.dylib`, which is, e.g., `libfoo.dylib.dSYM`. Hence the "for"